src/manifest/dbtProjectContainer.ts

```diff
@@ -23,7 +23,7 @@ export class DBTProjectContainer implements Disposable {
   private dbtWorkspaceFolders: DBTWorkspaceFolder[] = [];
   private _onManifestChanged = new EventEmitter<ManifestCacheChangedEvent>();
   public readonly onManifestChanged = this._onManifestChanged.event;
-  public currentSql?:string;
+  public currentSql?: string;

```

Comment:
AFAIK there was no need anymore for a global currentSql. As mentioned before, I would like to reduce the responsibilities of dbtProjectContainer.